modules/gatsby/src/templates/examples.jsx

```diff
@@ -1,9 +1,48 @@
-import React, {Component} from 'react'
-import styled from 'styled-components'
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { Link } from 'gatsby';
 
 import { graphql } from 'gatsby';
 
-import ExampleTableOfContents from '../components/layout/example-table-of-contents'
+import ExampleTableOfContents from '../components/layout/example-table-of-contents';
+
+const colors = {
```

Comment:
this is in preparation for a future where all of these variables will be in one space